kedge/core.yml

```diff
@@ -0,0 +1,13 @@
+name: core
```

Comment:
let's call it `wit` ? to make it easier to understand since the image name is `wit` :)

Reply:
true, fixed in https://github.com/fabric8-services/fabric8-wit/pull/1741/commits/ea3bbdcb14d6409ccf9b53d78364bd90e621ae21#diff-cb12c9593ee5832f1388ff84f6c7c18dR1